### PR TITLE
improved build payment req documentation

### PR DESCRIPTION
**Pull request by burdettadam**
Signed-off-by: Adam Burdett <adam@sovrin.org>

### CODE REVIEW

File: doc/data_structures.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ Example sources_json:
 ```
 
 ## method: indy_build_payment_req
-This API call is handled by LibSovToken build_payment_req_handler. 
+This API call is handled by LibSovToken build_payment_req_handler. Note all payment amounts in outputs must sum to the input utxo value.
```
Comment (brentzundel) [inline doc/data_structures.md]: This is not true unless: fees are 0 for the transaction; or we change the way fees are handled and they become an explicit part of the outputs.

File: doc/data_structures.md [hunk 2]
```diff
@@ -476,9 +476,13 @@ This API call is handled by LibSovToken build_payment_req_handler.
 
 [
     {
-        "address" : <str>,   // the payment address
-        "amount": <int>,    // the payment amount
+        "address" : <str>,   // the payment address correlating to the input utxo
```
Comment (brentzundel) [inline doc/data_structures.md]: At the plugin level, there is no requirement that one of the input addresses be used for whatever amount returns as change from the transaction. Apps that use libsovtoken through the libindy payments API may elect to follow this practice, but it should not be required.

Comment (cam-parra) [reply doc/data_structures.md]: @brentzundel could you please review?

Comment (brentzundel) [reply doc/data_structures.md]: I read your code and believe I understood it. 
It is certainly the case that a remainder address is required. This remainder address can be any valid address, it does not need to be one of the input addresses from the transaction.

**Comment by mattraffel** [comment]
Adam, its my understanding this PR is no longer needed.  Can you please close this PR so to clean up the outstanding PRs?  thanks

**Pull request closed without merging**